examples/cmd/vppagent-nsc/nsc.go

```diff
@@ -70,7 +70,7 @@ func main() {
 		vppAgentEndpoint: defaultVPPAgentEndpoint,
 	}
 
-	client, err := client.NewNSMClient(nil, nil)
+	clientList, err := client.NewNSMClientList(nil, nil)
```

Comment:
Why not calling the simple Client? Are you expecting the NSC will have to hanlde multiple connections?

Reply:
This nsc can be used to replace the ["standard"](https://github.com/networkservicemesh/networkservicemesh/blob/master/examples/cmd/nsc/nsc.go) nsc. It should be able to handle multiple "annotations", don't you think?

Reply:
Ehm, I think this is an example of a `smart` Client, i.e. one that does not use an external init Container. I am not sure if it is even possible that we can have an initContainer with MemIf.
That said - It can be perfectly fine having multiple connections as lomng as we find a way to demo/test it. For example, can we add this container in our tests under usecases? I know we barely test anythign related to MemIf today - only the Firewall is there. Every other test uses nsc and icmp-responders with Kernel interfaces.

Reply:
This mean this vppagent-nsc is not tested in the CI, since the usage of memif is hardcoded in it?